### PR TITLE
Feat openemr fix #7609 dashboard cards size consistency

### LINKED ISSUE #7609: feat: Change Patient Portal Layout to use tile format for mobile friendly design
As a patient I would like to be able to quickly and easily see the quick start dashboard on my phone, tablet, and desktop device. It would be helpful if there was an icon associated with each menu icon that I can retain in memory.  With a tile icon layout I would no longer need to use the menu navigation so that can be removed.

Design Goal: Make it easier for patients to quickly see their patient portal navigation items.

This builds on a lot of work and effort that Jerry Padgett has put into the patient portal to make it more user friendly.

### PR DESCRIPTION
**Pull request by adunsulag**
This fixes #7609.  This work has been sponsored by Open Plan IT and heavily builds upon what @sjpadgett has done with the portal single page app layout.

The issues referenced in the commit comments are for internal issues in the Open Plan IT project. 

Screenshots for the new design:
Desktop version:

Main home screen:

Image

Medical Reports Screen
Image

Settings Screen
Image

Dashboard Help Screen

Image

Medical Reports Help Screen
Image

Surface Pro:

Image

Ipad AIR

Image

iPhone XR

Image

Samsung Galaxy S8

Image

### CODE REVIEW

File: templates/portal/home.html.twig [hunk 1]
```diff
@@ -660,41 +678,260 @@
...
+                ,{
+                    title: "Appointments"|xl
+                    ,icon: "calendar-check"
+                    ,description: "View upcoming appointments and if allowed by your clinical staff make new appointments."|xl
+                    ,enabled: allow_portal_appointments
+                }
+                ,{
+                    title: "Make Payment"|xl
+                    ,icon: "wallet"
+                    ,description: ""|xl
```
Comment (bradymiller) [inline templates/portal/home.html.twig:735]: noted empty description

File: templates/portal/home.html.twig [hunk 2]
```diff
@@ -660,41 +678,260 @@
...
+            id: "help-card-reports-list"
+            ,backButton: true
+            ,backButtonCard: "reports-list-card"
+            ,backButtonClass: "reports-list-back-btn"
+            ,title: "Medical Reports Help"|xl
+            ,items: [
+                {
+                    title: "View Summary of Care"|xl
+                    ,icon: "notes-medical"
+                    ,description: "View your Summary of Care document that includes a printable version of your healthcare information.  This document can be used to transfer your care to another healthcare facility. In technical terms it is called a C-CDA."|xl
```
Comment (bradymiller) [inline templates/portal/home.html.twig:794]: don't use double spacing in translation constants

Comment (bradymiller) [reply templates/portal/home.html.twig:794]: also ensure this on other constants above and below

**Review by bradymiller — APPROVED** [review]

File: templates/portal/home.html.twig (hunk 1 above)
Comment (adunsulag) [reply templates/portal/home.html.twig]: Fixed

File: templates/portal/home.html.twig (hunk 2 above)
Comment (adunsulag) [reply templates/portal/home.html.twig]: Fixed

**Comment by adunsulag** [comment]
We talked on the Tuesday call about bringing this in so people can try this out on the demos.  If we don't like it the commit is pretty self-contained and we can revert it if needed.

**Pull request merged**